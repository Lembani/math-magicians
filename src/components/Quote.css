.quote {
  width: 50vw;
  margin: 0 auto;
  margin-top: 30vh;
  font-size: 1.6rem;
  text-align: center;
}
