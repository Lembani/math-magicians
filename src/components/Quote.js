import React from 'react';

import './Quote.css';

class Quote extends React.PureComponent {
  render() {
    return (
      <>
        <p className="quote">
          Mathematics is not about numbers, equations, computations or
          algorithms: it is about understanding. - William Paul Thurston
        </p>
      </>
    );
  }
}

export default Quote;
