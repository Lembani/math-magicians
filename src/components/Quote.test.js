import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Quote from './Quote';

test('paragraph', () => {
  render(<Quote />);
  const paragraph = document.querySelector('.quote');
  expect(paragraph).toBeInTheDocument();
});

it('It renders correctly', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
  