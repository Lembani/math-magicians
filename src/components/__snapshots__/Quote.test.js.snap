// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
<p
  className="quote"
>
  Mathematics is not about numbers, equations, computations or algorithms: it is about understanding. - William Paul Thurston
</p>
`;
